python-sdk/rebuff/sdk.py

```diff
@@ -29,22 +38,44 @@ class RebuffSdk:
     def __init__(
         self,
         openai_apikey: str,
-        pinecone_apikey: str,
-        pinecone_index: str,
-        openai_model: str = "gpt-3.5-turbo",
+        vector_db: VectorDB,
+        pinecone_apikey: Optional[str] = None,
+        pinecone_index: Optional[str] = None,
+        chroma_collection_name: Optional[str] = "rebuff",
+        openai_model: Optional[str] = "gpt-3.5-turbo",
     ) -> None:
-        self.openai_model = openai_model
         self.openai_apikey = openai_apikey
-        self.pinecone_apikey = pinecone_apikey
-        self.pinecone_index = pinecone_index
+        self.vector_db = vector_db
+
+        if self.vector_db.name == "PINECONE":
+            self.pinecone_apikey = pinecone_apikey
+            self.pinecone_index = pinecone_index
+
+        elif self.vector_db.name == "CHROMA":
```

Comment:
```suggestion
        if self.vector_db == VectorDB.PINECONE:
            self.pinecone_apikey = pinecone_apikey
            self.pinecone_index = pinecone_index

        elif self.vector_db == VectorDB.CHROMA:
```
Similar recommendation for within `initialize_vector_store()`.